Update/omot_024.txt

```diff
@@ -2362,17 +2362,17 @@ void main()
 		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f6d9a8>鷹野</color>", NULL, "<color=#f6d9a8>Takano</color>", NULL, Line_ContinueAfterTyping); }
 		ModPlayVoiceLS(4, 9, "ps3/s14/09/vt4bc_takano1029", 256, TRUE);
 		OutputLine(NULL, "「そして、私を支援するって話も…雛見沢症候群の研究を継続させるって甘い誘いも、全部なかったことになった！！」",
-				NULL, "\"And they didn't say a single word about supporting me... No heartfelt invitation to continue my Hinamizawa Syndrome research!!\"", Line_Normal);
+				NULL, "\"And they deserted all about supporting me... They canceled their heartfelt invitation to continue my Hinamizawa Syndrome research!!\"", Line_Normal);
 		ClearMessage();
 
 	//鷹野rvS14/09/VT4Bc_takano1030.「私がわざわざ大がかりなことをしなくても、勝手に権力が転がり込んできたんだから！！kvS14/09/VT4Bc_takano1031.　つまり…私を焚きつけてきたやつらは、最初から勢力争いが目的だったってわけよ！！」
 		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f6d9a8>鷹野</color>", NULL, "<color=#f6d9a8>Takano</color>", NULL, Line_ContinueAfterTyping); }
 		ModPlayVoiceLS(4, 9, "ps3/s14/09/vt4bc_takano1030", 256, TRUE);
 		OutputLine(NULL, "「私がわざわざ大がかりなことをしなくても、勝手に権力が転がり込んできたんだから！！",
-				NULL, "\"Even if I don't cause a major incident, all my power is being selfishly taken from me!!", Line_WaitForInput);
+				NULL, "\"Because before I cause a major incident, they were already able to seize power with little effort!!", Line_WaitForInput);
```

Comment:
```suggestion
				NULL, "\"Because even before I caused a major incident, they were already able to seize power with little effort!!", Line_WaitForInput);
```